src/main.ts

```diff
@@ -382,6 +383,12 @@ export async function createPipeline(config: Config): Promise<string> {
   if (typeof process.env.VIB_PUBLIC_URL === "undefined") {
     core.setFailed("VIB_PUBLIC_URL environment variable not found.")
   }
+  const verificationModeValues = ["PARALLEL", "SERIAL"]
```

Comment:
I think it is better if you move this to a constant in `constants.ts` and for consistency make it an enum.

Reply:
Done!